test/extended/testdata/application-template-dockerbuild.json

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
Don't copy this file, it already lives here: https://github.com/openshift/origin/blob/master/examples/sample-app/application-template-dockerbuild.json re-use that instead. We currently have 2 copies already, my goal is to go to only a single one, b/c it's easier to maintain.

Reply:
resolved